Makefile

```diff
@@ -4,15 +4,15 @@ all: poetry
 # Building
 # --------
 
-PYTHON_BIN   := python3.10
+PYTHON_BIN   := $(shell k-which-python)
```

Comment:
This command is only distributed with the `kup` install of K.

It'd be nice if the `Makefile` could automatically detect how K has been installed.